migrations/versions/1add42f9be06_add_period_model.py

```diff
@@ -0,0 +1,37 @@
+"""Add Period Model
+
+Revision ID: 1add42f9be06
+Revises: 57e3e1d9716a
+Create Date: 2022-12-21 13:53:33.115614
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1add42f9be06'
+down_revision = '57e3e1d9716a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('period',
+    sa.Column('zone', sa.Integer(), nullable=False),
+    sa.Column('is_online', sa.Boolean(), nullable=False),
+    sa.Column('start', sa.DateTime(), nullable=False),
+    sa.Column('end', sa.DateTime(), nullable=False),
+    sa.Column('duration', sa.Integer(), nullable=False),
```

Comment:
Does it make sense to store duration if you already storing `start` and `end`?

Reply:
probably not, more time is needed for a better solution